src/components/common-components/autocomplete/autocomplete.component.tsx

```diff
@@ -64,32 +91,35 @@ export function AutocompleteComponent<T>({
         />
       </div>
       {groupedOptions.length > 0 ? (
-        <ul
+        <div
           ref={tooltipRef}
           className={classNames("listbox")}
           style={styles.popper}
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
           onMouseDown={(getListboxProps() as any).onMouseDown}
         >
-          {groupedOptions.map((option, index) => (
-            <li
-              {...getOptionProps({ option, index })}
-              data-cy={option["data-cy"]}
-              onClick={(e: React.MouseEvent): void => {
-                e.stopPropagation();
-                setValue(option);
-              }}
-            >
-              <div className="container">
-                <div className="optionLabel">{getOptionLabel(option)}</div>
-                <div
-                  className="colorSample"
-                  style={{ backgroundColor: `#${getOptionColor(option)}` }}
-                />
-              </div>
-            </li>
-          ))}
-        </ul>
+          {groupedOptions.map((option, index) => {
+            if (option.name.trim() === "wolne") {
+              return <LabelComponent option={option} index={index} />;
+            }
+            return null;
+          })}
+          {groupedOptions.map((option, index) => {
+            if (option.name.trim() !== "wolne") {
```

Comment:
Lepiej tak nie robić, ponieważ nie mamy pewności tego, że ta zmiana będzie nazywać się w taki sposób. 
O wiele lepiej uzaleźnić to od stanu. 
![image](https://user-images.githubusercontent.com/21079319/111392180-54371d00-86b6-11eb-8dfa-0beac8011391.png)


I zrobić coś w stylu
```ts
const shiftTypes = useSelector(...)
....
shiftTypes[ShiftCode.W].name
```
`ShiftCode` - to zdefiniowany przez nas Enum zawierający często używane kody zmian

Reply:
No dobra.